### PR TITLE
Take off total pop race crosstabs

### PR DESCRIPTION
**Pull request by SashaWeinstein**
# Fix race crosstab of total population
This fix is in response to the fifth checkbox in [Amanda's long comment on the demographic indicator](https://github.com/NYCPlanning/db-equitable-development-tool/issues/80#issuecomment-1050981122). She correctly noticed that the `anh` percent was 100, when it should have been much lower. 

This information on percentage of total pop by race won't ultimately come from PUMS, it will come from the decennial census. We do use the info but only to calculate percentages, and it's reported in the denominator column. This means there is no good reason total pop by race should have it's own indicators with associated variances and percentages. 

## Changes to code
The changes are rather janky, just adding if statements to not iterate through races for total pop and then not including total pop in the logic to order the columns at the end. A better approach is to set total pop as a different sort of indicator that doesn't get crosstabbed by race. This make the code cleaner. But to hit deadlines a faster approach was taken 

### Question
Can we get total pop from decennial census? If so total pop can be removed as indicator here and a small but persistent headache goes away

### CODE REVIEW

File: aggregate/PUMS/aggregate_PUMS.py [hunk 1]
```diff
@@ -236,10 +237,22 @@ def add_category(self, indicator):
...
+            if ind_denom[0] != "total_pop"
+        ]
+        # Don't love hardcoding the beginning of this list, can be refactored
+        col_order = [
+            "total_pop-count",
+            "total_pop-count-cv",
+            "total_pop-count-moe",
+            "total_pop-pct",
+            "total_pop-pct-cv",
+            "total_pop-pct-denom",
```
Comment (td928) [inline aggregate/PUMS/aggregate_PUMS.py:252]: quick reference here to [one of Amanda's comment](https://github.com/NYCPlanning/db-equitable-development-tool/issues/80#issuecomment-1050981122) about leaving out the `total_pop-pct-denom` @SashaWeinstein

Comment (td928) [inline aggregate/PUMS/aggregate_PUMS.py:252]: quick reference here to [one of Amanda's comment](https://github.com/NYCPlanning/db-equitable-development-tool/issues/80#issuecomment-1050981122) about leaving out the `total_pop-pct-denom` @SashaWeinstein

Comment (SashaWeinstein) [reply aggregate/PUMS/aggregate_PUMS.py:252]: yep, you're right about that. good catch

**Comment by SashaWeinstein** [comment]
I think per our conversation at monday standup this can be further improved to not include total pop columns at all? Those come from the decennial census. I can work on that this morning, should be quick

**Comment by SashaWeinstein** [comment]
Ok new changes take away total pop as indicator, it won't be reported at all in final output. 

The only downside of the new code is that the total pop was good to double check denominators. We could make adding it optional for debug but maybe that's overkill

**Review by td928 — APPROVED** [review]

**Pull request merged**